html-forms/index.html

```diff
@@ -15,33 +15,26 @@ <h1>Coding Form!</h1>
       <br />
       <label for="select">Why is coding cool?</label>
       <br />
-      <select id="cool" name="'Why is coding cool?">
+      <select id="select" name="'Why is coding cool?">
         <option>Select an option</option>
         <option>Because it's interesting</option>
         <option>Because it's fun</option>
         <option>Because it's powerful</option>
       </select>
       <br />
-      <br />
-      <label for="radio">Favorite Language (choose one)</label>
-      <br />
-      <br />
+      <p>Favorite Language (choose one)</p>
       <input type="radio" id="html" name="language" />
       <label for="html">HTML</label>
       <input type="radio" id="css" name="language" />
       <label for="css">CSS</label>
       <input type="radio" id="javascript" name="language" />
       <label for="javascript">Javascript</label>
       <br />
+      <p>Would you like to sign up for more information?</p>
+      <input type="checkbox" />
+      <label for="checkbox">Sign me up!</label>
       <br />
-      <label for="checkbox"
-        >Would you like to sign up for more information?</label
-      >
-      <br />
-      <br />
-      <input type="checkbox" /> Sign me up!
-      <br />
-      <label for="comments">Comments</label>
+      <label for="comments" id="comments">Comments</label>
```

Comment:
The `id="comments"` needs to go on your `textarea`, not your `label`. The `for` attribute looks for an element with an `id` that matches it, and that's how we link them. By adding the `id` here, you're not linking it to the `textarea`

Reply:
Got it, I'll do some more reading